src/transformers/dependency_versions_table.py

```diff
@@ -0,0 +1,13 @@
+# THIS FILE HAS BEEN AUTOGENERATED. To update:
```

Comment:
Do we really have to generate this file? If it's just so that people can be quickly what versions need to be installed, I don't think it's worth adding another file to the lib. One can check the versions in `setup.py` which one usually does for python repos anyways no?

Reply:
But it might very much be possible that I'm not seeing something here - would be awesome if you can enlighten me @stas00 :-)

Reply:
That's the key feature of this PR - we need a table of dependencies available at run-time - since there is no setup.py once `transformers` is installed. How can we know which version to require if we don't have this data.

The add-no-new-file-way is to hardcode the required versions in both setup.py and wherever we do the run-time checking. Based on my experience - the two will not stay in sync unless someone is vigilantly always ensures they do.

This is absolutely possible, though. It's your call.

Reply:
An alternative would be to have `make style`/`make quality` be the commands that update/check consistency between the two lists.

Reply:
Good idea. Though no need to check, just running `python setup.py --name >> /dev/null 2>&1` is probably faster.

Reply:
Though it's silly to re-run something that 99.99% of the time will be doing nothing. but it's better than trying to remember.

Reply:
I'd also be in favor of putting it in `make style`

Reply:
Done.